osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs

```diff
@@ -116,7 +121,7 @@ private Drawable[] createContent(int index, ScoreInfo score)
                 new OsuSpriteText
                 {
                     Margin = new MarginPadding { Right = horizontal_inset },
-                    Text = score.DisplayAccuracy,
+                    Text = score.Accuracy.FormatAccuracy(alwaysShowDecimals: true),
```

Comment:
this shouldn't be happening. we added the `DisplayAccuracy` string for a reason.

Reply:
as mentioned elsewhere, please split these changes into a separate PR for further discussion. it's not clear why this is being done in this PR currently.